docs/Cliente REST/2-Axios.md

```diff
@@ -12,14 +11,18 @@ Para instalar digite os comandos abaixo:
 
 > yarn add axios
 
-## Entendo como funciona uma api 
+## O conceito de API para o Axios
+
+Primeiramente devemos comprender que o termo API é muito amplo e tem inumeras funcões. Sendo assim uma explicação mais simples para o axios seria que ele faz uma ponte entre API, ou seja, em um exemplo mais claro ele se torna um garço que deve trazer seu pedido de comida.
```

Comment:
Aqui está escrito garço, acho que seria garfo, não? Deve ter sido outro typo.

E tem funcões/inumeras, que poderia ser aplicada apenas a correção das palavras (funções/inúmeras)